pages/gallery.html

```diff
@@ -1,12 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Banana Gallery</title>
+    <link rel="stylesheet" href="../styles/style.css" />
+    <link rel="stylesheet" href="../styles/gallery.css" />
+  </head>
+  <body>
+    <div class="container">
+      <!-- <div class="header"> -->
+      <header>
+        <h1>Banana Gallery</h1>
+        <nav>
+          <ul>
+            <li><a href="../index.html">Home</a></li>
+            <li><a href="facts.html">Facts</a></li>
+            <li>Gallery</li>
+          </ul>
+        </nav>
+      </header>
+      <!-- </div> -->
+      <main>
+        <section class="img-container">
+          <div class="header-container">
+            <h2 id="body-header">Images</h2>
+          </div>
+          <a
+            href="https://www.eatthis.com/wp-content/uploads/sites/4/2021/07/bananas.jpg?quality=82&strip=all"
+            target="_blank"
```

Comment:
If using `target="_blank"` also consider adding `rel=”noopener”`, which will prevent the site being opened from having script access back to your page.